featureflags/evaluations/evaluator.py

```diff
@@ -293,8 +293,7 @@ def _check_prerequisite(self, parent: FeatureConfig,
                 log.info('Pre requisite flag %s should have the variations %s',
                          config.feature, pqs.variations)
 
-                if isinstance(variation, Unset) and variation.identifier \
-                        not in pqs.variations:
+                if variation.identifier not in pqs.variations:
```

Comment:
```suggestion
                if not isinstance(variation, Unset) and variation.identifier not in pqs.variations:
```
for me makes more sense because misconfiguration of flag using api can put invalid value and variation can be None then it will throw exception. We should look this in fututre to replace with `if variation and variation.identifier`

Reply:
Yeah thats a far better idea, i'll add this in!